ci/jenkins/test.sh

```diff
@@ -67,7 +68,8 @@ Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests o
         --testbed-type           The testbed type to run tests. It can be flexible-ipam, jumper or legacy.
         --ip-mode                IP mode for flexible-ipam e2e test. Default is $DEFAULT_IP_MODE. It can also be ipv6 or ds.
         --win-image-node         Name of the windows image node in containerd cluster. Images are built by docker on this node.
-        --kind-cluster-name      Name of the kind Cluster." 
+        --kind-cluster-name      Name of the kind Cluster
+        --ip-version             Specifies the ip-version for the kind cluster." 
```

Comment:
`ip-version` is a little confusing to me. I think naming it `enable-ipv6` would be better?